Project-Euler/test/Problem017.test.js

```diff
@@ -1,41 +1,11 @@
 import { countNumberWordLength } from '../Problem017.js'
 
 describe('Number letter count', () => {
-  const textFixture = [
-    {
-      name: 'should throw invalid input (NaN) error',
-      input: 'test',
-      error: 'Invalid input, please provide valid number'
-    },
-    {
-      name: 'should throw error if number < 1',
-      input: 0,
-      error: 'Please provide number greater that 1'
-    },
-    {
-      name: 'if the number provided is 5',
-      input: 5,
-      expected: 19
-    },
-    {
-      name: 'if the number provided is 100',
-      input: 100,
-      expected: 864
-    },
-    {
-      name: 'if the number provided is 1000',
-      input: 1000,
-      expected: 21124
-    }
-  ]
+  test.each([[5, 19], [100, 864], [1000, 21124]])('%i! = %i', (n, expected) => {
```

Comment:
Please use a sensible format such as `letters from 1 ... %i = %i`

Reply:
Addressed. Could you please check it.